src/components/SectionTitle/__tests__/SectionTitle.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import SectionTitle from '..';
+
+describe('SectionTitle component', () => {
+  const mockData = ['home', 'previous', 'current'];
+
+  it('renders correctly with data', () => {
+    const view = render(<SectionTitle path={mockData} />);
+    expect(view).toMatchSnapshot();
```

Comment:
Can we snapshot `view.baseElement` instead of just `view`?

Reply:
Can i use container instead? Container is more common for the project

Reply:
Updated